src/lib/pipeline.ml

```diff
@@ -84,9 +87,14 @@ type _ t =
   | Bam_pair: bam t * bam t -> bam_pair t
   | Somatic_variant_caller: somatic Variant_caller.t * bam_pair t -> vcf t
   | Germline_variant_caller: germline Variant_caller.t * bam t -> vcf t
+  | Rna_hla_typer: fastq_sample t -> hla_types t
 
 module Construct = struct
 
+  (* Consider:
+     1. The name of this type makes more sense as a noun, fastq_input.
+     2. `Paired_end should contain (File.t * File.t) list ... since they are paired.
+     3. There needs to be a simpler type of `Single_pair and `Paired_type renamed. *)
```

Comment:
This should be in an issue, not in comments.
(and “paired-end”/"single-end” seems to be the default nomenclature in bfx)